packages/web-components/src/components/va-alert/va-alert.scss

```diff
@@ -171,7 +171,9 @@ i.fa-times-circle::before {
 }
 
 :host(:not([uswds])[full-width='']) div.alert,
-:host(:not([uswds])[full-width='true']) div.alert {
+:host(:not([uswds])[full-width='true']) div.alert,
+:host(:not([uswds])[full-width=''][data-role="banner"]) div.alert,
+:host(:not([uswds])[full-width='true'][data-role="banner"]) div.alert {
```

Comment:
`data-role="banner"` is set on `va-alert` within the `va-banner` component: 

https://github.com/department-of-veterans-affairs/component-library/blob/1741-baner-alert-border/packages/web-components/src/components/va-banner/va-banner.tsx#L165-L172